pkgs/applications/video/mpv/scripts/mpris.nix

```diff
@@ -1,28 +1,37 @@
 { stdenv, fetchFromGitHub, pkgconfig, gobject-introspection, mpv }:
 
 stdenv.mkDerivation rec {
-  name = "mpv-mpris-${version}.so";
-  version = "0.4";
+  pname = "mpv-mpris";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "hoyon";
     repo = "mpv-mpris";
     rev = version;
-    sha256 = "1fr3jvja8s2gdpx8qyk9r17977flms3qpm8zci62nd9r5wjdvr5i";
+    sha256 = "07p6li5z38pkfd40029ag2jqx917vyl3ng5p2i4v5a0af14slcnk";
   };
 
   nativeBuildInputs = [ pkgconfig ];
 
   buildInputs = [ gobject-introspection mpv ];
 
+  # strip is not run as always in fixup phase because $out is a file and not a
+  # directory and mpris.so is not located in $out/lib. Hence, to make sure gcc
+  # is not referenced in the final output, we use `strip` like this instead of
+  # just copying it. See:
+  # https://discourse.nixos.org/t/debug-why-a-derivation-has-a-reference-to-gcc/7009
   installPhase = ''
-    cp mpris.so $out
+    mkdir -p $out/share/mpv/scripts
+    cp mpris.so $out/share/mpv/scripts
```

Comment:
You can just use

```suggestion
    install -Dt $out/share/mpv/scripts mpris.so
```

Reply:
I used `install -Dt` upstream: https://github.com/hoyon/mpv-mpris/pull/38